packages/protocol/scripts/make-release-6-changes.ts

```diff
@@ -0,0 +1,35 @@
+import { ensureLeading0x, trimLeading0x } from '@celo/base'
+import { readJsonSync, writeJsonSync } from 'fs-extra'
+import {
+  getProposedProxyAddress,
+  isProxyRepointAndInitForIdTransaction,
+} from '../lib/compatibility/verify-bytecode'
+import { ProposalTx } from './truffle/make-release'
+
+const makeRelease3Changes = (releaseProposal: ProposalTx[]) => {
```

Comment:
Typo: release 6

Reply:
Or something different given the naming discussion above, both for the function name and filename.